raiden/tests/integration/fixtures/raiden_network.py

```diff
@@ -85,7 +96,8 @@ def raiden_chain(
         )
 
     exception = RuntimeError('`raiden_chain` fixture setup failed, token networks unavailable')
-    with gevent.Timeout(seconds=30, exception=exception):
+    timeout = 120 if blockchain_type == 'parity' else 30
```

Comment:
Since 

`timeout = 120 if blockchain_type == 'parity' else 30`

is used in multiple places in these tests I would suggest putting it in a function along with a docstring explaining why the choice of different timeouts per client